File: 25.10.22 牛客2025秋季算法编程训练联赛2-基础组/2. 牛子616.cpp
#include <iostream>
#include <string>
#include <unordered_map>

using namespace std;

int main() {
    int n;
    string a;
    
    cin >> n;
    cin >> a;
    
    unordered_map<int, int> dic;
    
    for (char c : a) {
        if (isdigit(c)) {
            int digit = c - '0';
            dic[digit]++;
        }
    }
    
    int count6 = (dic.find(6) != dic.end()) ? dic[6] : 0;
    int count1 = (dic.find(1) != dic.end()) ? dic[1] : 0;
    
    if (count6 > count1) {
        cout << count1 << endl;
    } else {
        cout << count6 - 1 << endl;
    }
    
    return 0;
}

File: 25.10.22 牛客2025秋季算法编程训练联赛2-基础组/2. 牛子616.py
n=int(input())
a=input()
dic={}
ans=0
for i in a:
    if int(i) in dic:
        dic[int(i)]+=1
    else:
        dic[int(i)]=1
print(dic)
if dic[6]>dic[1]:
    print(dic[1])
else:
    print(dic[6]-1)